src/Altinn.Apps/AppTemplates/AspNet/App/Program.cs

```diff
@@ -16,7 +18,25 @@ public class Program
         /// <param name="args">The command line arguments used when starting the application.</param>
         public static void Main(string[] args)
         {
-            CreateHostBuilder(args).Build().Run();
+            TelemetryConfiguration telemetryConfiguration = TelemetryConfiguration.CreateDefault();
+            telemetryConfiguration.InstrumentationKey =
+                    System.Environment.GetEnvironmentVariable("ApplicationInsights__InstrumentationKey") ?? string.Empty;
```

Comment:
How would this work in a local dev environment? Do we get any error messages or does the app simply start up with an invalid telemetryClient ?

Reply:
It just starts up with an uninitialized telemetryclient (won't send any events). It is also part of the acceptance criteria that missing information should not crash the app.